src/content-helper/dashboard-page/pages/traffic-boost/sidebar/components/tabs-content.tsx

```
@@ -77,12 +78,31 @@ export const TabsContent = ( {
+				escapeHTML(
+					/* translators: %1$s: Lobby Post link, %2$s: VIP Documentation link */
+					__(
+						'%1$s is currently not enabled for your Site ID. Information about requesting access to Traffic Boost can be found %2$s.',
+						'wp-parsely'
+					)
+				),
+				lobbyLink,
+				docsLink
+			);
+
+			errorMessage = <ContentHelperErrorMessage children={ message } />;
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)